src/cdk/testing/protractor/protractor-element.ts

```diff
@@ -78,7 +78,8 @@ export class ProtractorElement implements TestElement {
   }
 
   async clear(): Promise<void> {
-    return this.element.clear();
+    await this.element.sendKeys(Key.BACK_SPACE);
```

Comment:
should the `sendKeys` call be moved into the `clearElement` method?

Reply:
`clearElement` is not called by `ProtractorElement` afaiu, so it won't help unfortunately.

Reply:
Ah my bad, I was confused which one I was looking at